File: Examples/test-suite/ocaml/typedef_classforward_same_name_runme.ml
open Swig
open Typedef_classforward_same_name

let _ =
  let foo = new_Foo '() in
  ignore (foo -> "[x]" (5));
  assert (_extractFoo '(foo) as int = 5);
  let boo = new_Boo '() in
  ignore (boo -> "[x]" (5));
  assert (_extractBoo '(boo) as int = 5)
;;

File: Examples/test-suite/lua/typedef_classforward_same_name_runme.lua
local v=require("typedef_classforward_same_name")

local foo = v.Foo()
foo.x = 5
assert(v.extractFoo(foo) == 5)
local boo = v.Boo()
boo.x = 5
assert(v.extractBoo(boo) == 5)
